### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by titu**
Routing implemented for the specified end-points

**NOTE**: this PR contains the changes made in the past [PR for DB Connect](https://github.com/titu/canvas-consumer/pull/4)

### CODE REVIEW

File: routes/router.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const api = require('../api');
+
+let initializeRouter = (app) => {
+    app.get('/', (request, response) => {
+        response.send('Welcome to Canvas Consumer API.');
+    });
+    app.post('/link-account', api.linkAccount);
```
Comment (arealmaas) [inline routes/router.js]: If we keep every routing-logic here, it's gonna be a huge file in the end. How about moving all the app.post('link-ccount') etc inside the api? And just use courseApi.init(app) or something here. Then we have control over the API-path in the api-files as well.

**Review by arealmaas — CHANGES_REQUESTED** [review]

File: routes/router.js (hunk 1 above)
Comment (titu) [reply routes/router.js]: 💯 👍

File: api/courses.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const helper = require('../helper');
+const responseHelper = helper.response;
+
+let init = (app) => {
+    app.get('/courses', getCourses);
+};
+
+let getCourses = (request, response) => {
```
Comment (arealmaas) [inline api/courses.js:8]: beautiful <3

**Review by arealmaas — APPROVED** [review]

File: api/courses.js (hunk 2 above)
Comment (titu) [reply api/courses.js:8]: thank you :)

**Pull request merged**